.github/workflows/nix-action-default.yml

```diff
@@ -284,6 +284,73 @@ jobs:
       name: Building/fetching current CI target
       run: NIXPKGS_ALLOW_UNFREE=1 nix-build --no-out-link --argstr bundle "default"
         --argstr job "hydra-battles-single"
+  build-doc:
+    needs:
+    - hydra-battles-single
+    runs-on: ubuntu-latest
+    steps:
+    - name: Determine which ref to test
+      run: "if [ ${{ github.event_name }} = \"push\" ]; then\n echo \"tested_ref=${{\
+        \ github.ref }}\" >> $GITHUB_ENV\nelse\n merge_commit=$(git ls-remote ${{\
+        \ github.event.repository.html_url }} refs/pull/${{ github.event.number }}/merge\
+        \ | cut -f1)\n if [ -z \"$merge_commit\" ]; then\n echo \"tested_ref=refs/pull/${{\
+        \ github.event.number }}/head\" >> $GITHUB_ENV\n else\n echo \"tested_ref=refs/pull/${{\
+        \ github.event.number }}/merge\" >> $GITHUB_ENV\n fi\nfi\n"
+    - name: Git checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        ref: ${{ env.tested_ref }}
+    - name: Cachix install
+      uses: cachix/install-nix-action@v12
+      with:
+        nix_path: nixpkgs=channel:nixpkgs-unstable
+    - name: Cachix setup coq
+      uses: cachix/cachix-action@v8
+      with:
+        name: coq
+    - name: Cachix setup coq-community
+      uses: cachix/cachix-action@v8
+      with:
+        authToken: ${{ secrets.CACHIX_AUTH_TOKEN }}
+        name: coq-community
+    - name: Cachix setup math-comp
+      uses: cachix/cachix-action@v8
+      with:
+        name: math-comp
+    - name: Build Alectryon movies
+      run: |
+        nix-shell --run "cachedMake"
+        nix-shell --run "make -C doc/movies -j $(nproc) all"
+    - name: Build coqdoc
+      run: nix-shell --run "make -j $(nproc) html"
+    - name: Compile LaTeX document
+      run: |
+        nix-shell --run "make pdf"
+        # I need to run make pdf twice to get reference resolved :(
+        touch doc/hydras.tex
+        nix-shell --run "make pdf"
```

Comment:
I run `make pdf` twice to work around the unresolved reference issue. There is probably a better fix (which probably involves fixing a makefile). Suggestions welcome!

Reply:
@Zimmi48 Do you still need to run `make pdf`twice ? I commited the move of `bibtex hydras` upwards.

Reply:
Cool! I'll have a look tonight.

Reply:
Indeed, it works now!